index.html

```diff
@@ -15,11 +15,12 @@
       <p>Use the left and right arrow to control the car!</p>
       <img src="./images/arrows.png" alt="" class="arrows-img" />
     </div>
-
     <div id="game-board">
-      <canvas id="canvas" width="500" height="700"></canvas>
+      <canvas id="canvas" width="500" height="700" style="border: 0cm;"></canvas>
+      <img src="/images/road.png" alt="" id="road-img" />
+      <img src="/images/car.png" alt="" id="car-img"/>
     </div>
-
-    <script type="text/javascript" src="js/index.js"></script>
+    <script src="js/scripts/main.js"></script>
+    <script type="text/javascript" src="js/scripts/index.js"></script>
```

Comment:
Remember that you don't have a index.js file anymore, you changed it to Game. So you're going to need to update it here as well